application/validation.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Validation
+  POSITIVE_INFINITY = +1.0 / 0.0
+
+  def name_validation(name)
```

Comment:
move validation methods from module to separate classes
```
name_validator = NameValidator.new(name)
name_validator.valid? # returns true/false
name_validator.errors # returns an array of errors
```

Reply:
Created validating classes for name, password, login and age.